src/backend/providers/playnite/PlayniteProvider.cpp

```diff
@@ -229,7 +228,8 @@ bool PlayniteProvider::validate_game(const PlayniteGame& game, const PlayniteCom
         return false;
     }
 
-    if (game.play_action.type.isEmpty()) {
+    const PlayniteGameLaunchInfo::LaunchType& launch_type = PlayniteLaunchHelper::get_launch_type_enum(game.play_action.type);
```

Comment:
`PlayniteLaunchHelper::get_launch_type_enum` returns a plain type, no need for reference.